design/design-doc-stacks.md

```diff
@@ -301,24 +386,25 @@ Possible solutions for package processing are listed below, along with their pro
   * Package images only contain the package contents, no processing logic is included in the package
   * The SM starts a job pod with an init container for the package image and copies all its contents to a shared volume in the job pod.  The Crossplane package processing logic runs in the main pod container and runs over the shared volume, sending all artifacts to `stdout`.
   * PRO: Package images are significantly smaller since they will only contain yaml files and icons, no binaries or libraries.
+  * PRO: The processing/unpacking logic can easily move to its own image that can be used as a base layer in the future (or a CLI tool), so this approach is not very divergent.
   * CON: The SM needs to understand package content and layout.
 * CLI tool
   * The unpacking logic could be built into a CLI tool that can be run over a package image.
```

Comment:
I think I am missing something about what `unpacking` means . . . sorry if this was already explained elsewhere in the document. My question is: what is the step that happens after unpacking? Does anything need to be run? Does this mean that the tool would unpack the bundle and then `apply` the `install.yaml`, or something along those lines?

Reply:
1. Install CRDs
1. Generate basic RBAC for controller and include permissions from rbac.yaml
1. Install RBAC
1. Install the install.yaml for the controller

Example rbac.yaml from gitlab example:
```yaml
rules:
- apiGroups:
    - controller.gitlab.com
  resources:
    - gitlabs
    - gitlabs/status
  verbs:
    - "*"
- apiGroups:
    - ""
  resources:
    - secrets
  verbs:
    - get
    - list
    - watch
    - create
    - update
- apiGroups:
    - cache.crossplane.io
    - compute.crossplane.io
    - core.crossplane.io
    - storage.crossplane.io
    - aws.crossplane.io
    - cache.aws.crossplane.io
    - azure.crossplane.io
    - gcp.crossplane.io
    - workload.crossplane.io
  resources:
    - "*"
  verbs:
    - "*"
```

I don't see an example here for rbac.yaml so I'm just restating rbac as I knew it in a previous version. I think just having the inner rbac rules instead of service account and role bindings would be safer for the privileged stack installer to generate and manage the service account and binding instead of it being installed as part of the package. Likely it's a nicer user experience to not worry about this plumbing, and also easier to audit what the package needs for permissions.

Reply:
Included a terminology item for `unpacking` and added an `rbac.yaml` example in b4f825b

We may want to revisit this when we expand on the concepts of App Stacks and Infra Stacks where the rbac.yaml will likely differ.